Packs/CommonScripts/Scripts/UnEscapeIPv6/UnEscapeIPv6.py

```diff
@@ -0,0 +1,38 @@
+import demistomock as demisto
+
+
+beginning_characters_list = ['(', '[', ' ', '-', '\'', '"', '.', ',', '`']
+ending_characters_list = [')', ']', ' ', '-', '\'', '"', '.', ',', '`']
+
+
+def clearing_the_address_from_unrelated_characters(ipv6_address: str) -> str:
```

Comment:
I think you need to make better logic here since a string could be: `[({::1234})]`, in this case, you are only trimming the `[,]` characters and not all the characters you wanted to.

Reply:
As in the previous comment